CustomKnight/Skin/Base/SkinManager.cs

```diff
@@ -114,6 +114,68 @@ public static class SkinManager{
             {"Charm_40_5",new Charm("Charm_40_5",40)}
            // {"PinsScarab", new Pins()}
         };
+        public static Dictionary<string, Skinable> InvSkinables = new Dictionary<string, Skinable> {
```

Comment:
why did you make this a separate Dictionary? seems to be the cause of a lot of duplicate code?